src/granite/error.cr

```diff
@@ -5,10 +5,6 @@ class Granite::Error
   end
 
   def to_s
-    if @field == :base
-      @message
-    else
-      "#{@field.to_s.capitalize} #{message}"
-    end
+    (@field == :base) ? @message : "#{@field.to_s.capitalize} #{message}"
```

Comment:
Can we leave this as it was? I don't see the benefit to the more subtle syntax.

Reply:
I can change it back if you want.  I just have a thing for ternary, easier to read and cleaner imo.

Reply:
I guess this is the sort of thing that a style guide helps communicate. My preference is to avoid ternaries wherever possible. 

More than anything, this is an unrelated change to the rest of the pull. I don’t feel really strong about changing this, but I’d like to avoid someone else changing it back simply because they prefer it to be the other way. It just creates more potential for bugs to sneak in and opens the door for endless bike shedding.

Reply:
Fair enough.